Mathlib/Algebra/Algebra/TransferInstance.lean

```diff
@@ -23,23 +22,26 @@ variable (R) in
 /-- Transfer `Algebra` across an `Equiv` -/
 protected abbrev algebra (e : α ≃ β) [Semiring β] :
     let _ := Equiv.semiring e
-    ∀ [Algebra R β], Algebra R α := by
-  intros
-  letI : Module R α := e.module R
-  fapply Algebra.ofModule
-  · intro r x y
-    change e.symm (e (e.symm (r • e x)) * e y) = e.symm (r • e.ringEquiv (x * y))
-    simp only [apply_symm_apply, Algebra.smul_mul_assoc, map_mul, ringEquiv_apply]
-  · intro r x y
-    change e.symm (e x * e (e.symm (r • e y))) = e.symm (r • e (e.symm (e x * e y)))
-    simp only [apply_symm_apply, Algebra.mul_smul_comm]
+    ∀ [Algebra R β], Algebra R α := fast_instance%
+  letI := Equiv.semiring e
+  letI := e.smul R
+  { algebraMap := e.ringEquiv.symm.toRingHom.comp (algebraMap R β)
```

Comment:
I think you probably want
```suggestion
  { algebraMap := 
    { toFun r := e.symm (algebraMap R β r)
      __ := e.ringEquiv.symm.toRingHom.comp (algebraMap R β) }
```
which will ensure that it reduces to `e.symm (algebraMap R β r)` under `with_reducible`